src/components/textarea/textarea.stories.ts

```diff
@@ -0,0 +1,89 @@
+import { select, text, number } from "@storybook/addon-knobs";
+import { boolean, storyFilters } from "../../../.storybook/helpers";
+import readme from "./readme.md";
+import { html } from "../../../support/formatting";
+
+export default {
+  title: "Components/Textarea",
+  parameters: {
+    notes: readme
+  },
+  ...storyFilters()
+};
+
+export const simple = (): string => html`
+  <div style="width:300px;text-align:center;">
+    <calcite-textarea
+      scale="${select("scale", ["s", "m", "l"], "m")}"
+      placeholder="${text("placeholder", "Add Notes")}"
+      ${boolean("disabled", false)}
+      cols="${number("cols", 20)}"
+      rows="${number("rows", 2)}"
+      ${boolean("resize-disabled", true)}
+      ${boolean("horizantal-resize-disabled", false)}
+      ${boolean("vertical-resize-disabled", false)}
+      ${boolean("required", false)}
+      label="${text("label", "")}"
+      name="${text("name", "")}"
+    >
+    </calcite-textarea>
+  </div>
+`;
+
+export const withSlottedElements = (): string => html`<div style="width:300px;max-width:100%;">
+<calcite-textarea
+scale="${select("scale", ["s", "m", "l"], "m")}"
+placeholder="${text("placeholder", "Add Notes")}"
+maxlength="${number("maxlength", 50)}"
+>
+<calcite-button  slot="${text("slot", "footer-leading")}">RESET</calcite-button>
+</caclite-textarea>
+</div>`;
+
+export const disabled = (): string => html`<div style="width:300px;max-width:100%;">
+<calcite-textarea
+disabled
+>
+</caclite-textarea>
+</div>`;
+
+export const readonly = (): string => html`<div style="width:300px;max-width:100%;">
+<calcite-textarea
+readonly
+>
+</caclite-textarea>
+</div>`;
+
+export const resizeDisabled = (): string => html`<div style="width:300px;max-width:100%;">
```

Comment:
The `resizeDisabled`, `readonly`, and `disabled` stories can all be TestOnly. You can think of it like anything that a user can achieve by using knobs in the `simple` or other stories doesn't need to be repeated in the public stories.

So stuff like adding/removing properties can be TestOnly, but adding stuff to slots or other things that can't be done using knobs warrent another public story